tests/core/pyspec/eth2spec/test/utils/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .utils import (
+    vector_test,
+    with_meta_tags,
+    build_transition_test,
+)
+
+
+__all__ = [  # avoid "unused import" lint error
+    "vector_test",
+    "with_meta_tags",
+    "build_transition_test",
+]
```

Comment:
moved `utils.py` as the one file to a module `utils`, that re-exports the existing stuff in this `__init__.py` and adds the helper code for the random tests into `utils/random.py`